lib/afinter.c

```diff
@@ -582,7 +582,8 @@ afinter_message_posted(LogMessage *msg)
 
       stats_lock();
       StatsClusterKey sc_key;
-      stats_cluster_logpipe_key_legacy_set(&sc_key, SCS_GLOBAL, "internal_source", NULL );
+      stats_cluster_logpipe_key_set(&sc_key, "internal_source", NULL, 0);
+      stats_cluster_logpipe_key_add_legacy_alias(&sc_key, SCS_GLOBAL, "internal_source", NULL );
```

Comment:
[optional]

It would be nice, if the internal source counters could be `internal_events_total` with a sensible sum. We would need a `written` or something analogous counter to make it work, and as this is an experimental feature and internal source is not top prio ATM, I think we can keep it like this for now.

Reply:
We can do this later.